content/en/blog/2023/istioday-kubecon-eu/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Join us for IstioDay at KubeCon Europe 2023!"
+description: The first ever IstioDay at KubeCon will take place in April.
+publishdate: 2023-01-26
+attribution: "Istio Steering Committee"
+keywords: [IstioCon,Istio,conference]
+---
+
+IstioDay Europe 2023, set for April 18, will be the first ever conference for Istio, the industry’s [most popular service mesh](https://www.cncf.io/wp-content/uploads/2020/11/CNCF_Survey_Report_2020.pdf), collocated with the KubeCon. Istio, which is currently [a graduating project at CNCF](https://www.cncf.io/projects/istio/) will mark its presence at the KubeCon in 2023, connecting community members across the globe with Istio’s ecosystem. Visit the [conference website](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/cncf-hosted-co-located-events/istio-day/) for all the information related to the event.
```

Comment:
Have we mentioned we are trying to graduate at this point?

Reply:
No, it was a mistake. fixed.